entrypoint.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```

Comment:
Would you mind renaming `entrypoint.sh` to `docker-entrypoint.sh` to communicate more clearly what the intention of the file is

Reply:
entrypoint.sh is a common naming for docker development, but I can rename it.